<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherLogsPage</class>
 <widget class="QWidget" name="OtherLogsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>538</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string notr="true">Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QLineEdit" name="searchBar"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="findButton">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QPlainTextEdit" name="text">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QPushButton" name="btnCopy">
           <property name="toolTip">
            <string>Copy the whole log into the clipboard</string>
           </property>
           <property name="text">
            <string>&amp;Copy</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="btnDelete">
           <property name="toolTip">
            <string>Clear the log</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="btnPaste">
           <property name="toolTip">
            <string>Upload the log to paste.ee - it will stay online for a month</string>
           </property>
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="btnClean">
           <property name="toolTip">
            <string>Clear the log</string>
           </property>
           <property name="text">
            <string>Clean</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btnReload">
           <property name="text">
            <string>Reload</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QComboBox" name="selectLogBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>selectLogBox</tabstop>
  <tabstop>btnReload</tabstop>
  <tabstop>btnCopy</tabstop>
  <tabstop>btnPaste</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnClean</tabstop>
  <tabstop>text</tabstop>
  <tabstop>searchBar</tabstop>
  <tabstop>findButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
